solution/ui/e2e/cypress/support/common-commands/login.js

```diff
@@ -4,10 +4,11 @@ export const eregsLogin = ({
     password,
     landingPage = "/policy-repository/",
 }) => {
-    cy.visit(landingPage);
+    cy.visit('/admin');
     cy.wait(1000);
     cy.get("#id_username").type(username);
     cy.get("#id_password").type(password);
     cy.get("#login-form").submit();
+    cy.visit('/policy-repository/');
 };
 
```

Comment:
since we no longer can visit /policy-repository/ and sign in with django admin, to get around it we will visit /admin and sign in that way but it would redirect to /admin so I updated this to then visit /policy-repository.

Reply:
Line 12 should still be `cy.visit(landingPage);`.  If you look at like 5, `eregsLogin` takes an object as a parameter, and one of the fields in that object is `landingPage`.  We're providing `/policy-repository/` as a default value for `landingPage`.  So you can just visit `landingPage` if you want to go visit the policy repository.

Reply:
The issue is since we do not have a way to mock signing in with EUA it wont get redirected back to policy repository. But I see now we also can sign in from the left nav, which then means we should redirect back to the left nav. Best solution is to pass a parameter where it should go back for cypress to know. Since we are backdoor signing in.